crates/common/consensus/src/electra/beacon_state.rs

```diff
@@ -91,12 +92,42 @@ use crate::{
     withdrawal::Withdrawal,
     withdrawal_request::WithdrawalRequest,
 };
+pub mod quoted_u8_var_list {
```

Comment:
```suggestion

pub mod quoted_u8_var_list {
```

A new line is missing here